archive/d/dg/file-io.dg

```diff
@@ -0,0 +1,22 @@
+import '/os'
+
+write_file = filename s ->
+    except
+        err =>
+            open filename 'w' |>.write <| s
+        err :: OSError =>
+            print "Cannot open file" filename
+
+read_file = filename ->
+    except
+        err =>
+            with fd = open filename 'r' =>
+                print $ fd.read!
+        err :: OSError =>
+            print "Cannot open file" filename
+
+
+filename = "tmp_file"
+write_file filename "Example text"
+read_file filename
+os.remove filename
```

Comment:
I don't believe we remove the file typically. Your solution is probably cleaner, but it's not consistent with the other solutions. We can change those if you feel strongly about your implementation. 👍